types/tsconfig.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "../tsconfig.json"
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {"@testing-library/dom": ["."]}
+  }
```

Comment:
These config changes seem unrelated, unless you're planning on pushing more files to this PR

Reply:
this config is useful because types tests are now using the types from `node_modules` instead of using types from their folder. I you made some changes to types the tests will fail without it because is used the alias `'@testing-library/dom'` to import types instead of relative path